swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -56,26 +55,13 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
 
 ### Release channels
 
-Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
-
-Ballerina distribution releases strictly follow [SemVer](https://semver.org/) with major.minor.patch version numbers.
+Ballerina distributions of the Swan Lake channel are currently released as preview releases. 
```

Comment:
Still we have 2 channels (Swan Lake and 1.x channel)
Still we will be doing patch releases to 1.x channel